test/common/EdgeModule.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace common
+{
+    using System;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public enum EdgeModuleStatus
```

Comment:
Use Microsoft.Azure.Devices.Edge.Agent.Core.ModuleStatus

Reply:
I wanted to use an existing enum, but ModuleStatus includes values Unknown, Backoff, Unhealthy, and Failed, which are not valid here. Also this test code doesn't interop with any other code that uses ModuleStatus so it didn't seem appropriate.